noughts/lib/noughts/application.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Noughts.Application do
+  use Application
+
+  def start(_type, _args) do
+    children = [
+      Noughts.Stash,
+      Noughts.Controller,
+      Noughts.SubSupervisor
+    ]
+    
```

Comment:
If the stash crashes, can the other processes continue, or should they be restarted.

If it's OK to continue, then one_for_one is OK.